ansible/configs/rhel8lab/post_software.yml

```diff
@@ -44,5 +44,30 @@
   tags:
     - post_flight_check
   tasks:
+
+- name: Print Console URL
+  agnosticd_user_info:
+    msg: "{{ item }}"
+  loop:
+    - "You can access your bastion (workstation) via SSH:"
+    - "ssh cloud-user@{{ public_ip_address }}"
+    - ""
+    - "Make sure you use the username 'cloud-user' and the password 'r3dh4t1!' when prompted."
+    - ""
+    - "For reference, the data you need to create your clouds.yaml file is:"
+    - ""
+    - "clouds:"
+    - "  {{ osp_project_name }}:"
+    - "    auth:"
+    - "      auth_url: {{ osp_auth_url }}"
+    - "      username: {{ guid }}-user"
+    - "      project_name: {{ osp_project_name }}"
+    - "      project_id: {{ hostvars['localhost']['osp_project_info'][0].id }}"
+    - "      user_domain_name: Default"
+    - "      password: {{ hostvars['localhost']['heat_user_password'] }}"
+    - "    region_name: regionOne"
+    - "    interface: public"
+    - "    identity_api_version: 3"
+  
```

Comment:
Indent level is wrong, this should be a task under `tasks:`, same level of indentation as `- debug:` below.